README.md

```diff
@@ -1 +1,13 @@
-# msgraph-github-content-utility
\ No newline at end of file
+# msgraph-github-content-utility
+## Description
+A .NET Core utility library that allows for ease of connecting to a GitHub repository using a registered GitHub app and an RSA Private Key.
+
+## Features
+This library will allow an authenticated GitHub app to do the below actions.
+1. Read blob contents from the repository.
+1. Write blob contents to the repository.
```

Comment:
numbering is incorrect

Reply:
Good ol' GitHub automatically renders it correctly for you. This helps in preventing potential miss-numbering when someone inserts a new point in between two other numbered points, or just plain old human error. 🙂

![image](https://user-images.githubusercontent.com/40403681/87957510-08958a80-cab9-11ea-8526-fb2af77cb709.png)
https://github.com/microsoftgraph/msgraph-github-content-utility/tree/is/project-init

Reply:
I find this style easier to maintain as well.